### PR TITLE
ci: fixed currency bot generator

### PR DESCRIPTION
**Pull request by kirrg001**
refs https://github.com/instana/nodejs/commit/f65e4fa4feb551eb0e9c378e04b39b2fe3ab78ae#diff-2f6a6fc5522f65f359ea21126fc9aa885f0688d5b0eb6c864f647d9bcdae6e23R868

### CODE REVIEW

File: bin/currency/generate-currency-report.js [hunk 1]
```diff
@@ -89,8 +89,16 @@ currencies = currencies.map(currency => {
...
+
+    // NOTE: Possible values are 0-days, 45-days or Deprecated
+    if (currency.policy === 'Deprecated') {
       upToDate = true;
+    } else {
+      const policyDays = currency.policy.match(/(\d+)-/);
+
+      if (policyDays && Number(daysBehind) < Number(policyDays[1]) && diff !== 'major') {
+        upToDate = true;
+      }
```
Comment (kirrg001) [inline bin/currency/generate-currency-report.js:101]: @aryamohanan Putting it here.

Can we remove the "deprecated" column from currencies.json now?
Seems like we don't need this column anymore if we switch to policy -> "Deprecated".

Comment (kirrg001) [reply bin/currency/generate-currency-report.js:101]: Cool. Will you raise a PR? Thanks!

Comment (aryamohanan) [reply bin/currency/generate-currency-report.js:101]: Sure

**Review by aryamohanan — APPROVED** [review]

File: bin/currency/generate-currency-report.js (hunk 1 above)
Comment (aryamohanan) [reply bin/currency/generate-currency-report.js:101]: https://github.com/instana/nodejs/pull/1439/files

**Review by abhilash-sivan — APPROVED** [review]

**Pull request merged**